examples/leg_example.cpp

```diff
@@ -7,6 +7,13 @@
  * This example demonstrates how you can use the mjbots_control_loop and the cartesian leg to create a hopping robot
  */
 
+
+//TODO DEPRECATED: NO LONGER WORKS WITH THE JOINT CLASSES
```

Comment:
why is this broken? You shouldn't be breaking old code. Like you should still be returning vectors of floats. Maybe its just an issue with the constructor? Either way must be fixed before its merged in.